models/resources.ts

```diff
@@ -28,3 +28,10 @@ export interface Service {
   yTwitter: string
   Communities: string[]
 }
+
+export interface ServicesLists {
+  Name: string
+  Services: string[] // list of IDs linking to services table
+  Status: string
+  Owner: string[] // list of IDs linking to organizations table
+}
```

Comment:
This I think we might need to iterate, we can go through v1 maybe in tomorrows meeting. Some points we might consider adding... 
* List_Id (UUID) as a unique identifier as well as the name
* Description (String) 
* Categories (String) but are kinda Enumy 
* Maybe Primary Owner vs Other Owners so we can at least tie the list to a primary owner

Reply:
`ListId` and `Description` make sense to me.
- `Categories` also makes sense but wouldn't it be redundant because that info would be contained in the linked services? It might be a question of read vs write speed because if the categories are an explicit field we can display them faster but when we allow people to add their own services, we have to write that additional info. It's probably negligible either way in terms of speed though so maybe we go with whatever makes the code simpler?
- What would something like `PrimaryOwner` vs `OtherOwners` be? My assumption was that `Owner` would just be the person / org that uploaded the the list.

For reference, these models are basically just the fields from [the auto-generated Airtable API docs](https://airtable.com/appivaPwNjH0Rs2RT/api/docs#javascript/table:services%20lists:fields).

Reply:
Yeah I agree on the Categories point. Figured we just might run into some issues if we have many owners for a list if we need to do admin work on it